apps/studio/src/background/lib/electron/ProtocolBuilder.ts

```diff
@@ -78,6 +79,11 @@ export const ProtocolBuilder = {
       const normalized = path.normalize(pathName)
       const fullPath = path.join(platformInfo.userDirectory, "plugins", normalized)
       log.debug("resolving", pathName, 'to', fullPath)
+      console.log("plugin", pathName, fullPath, {host:url.host})
+      if (bksConfig.get(`plugins.${url.host}.enabled`) === false) {
```

Comment:
How do I identify the name of the plugin to set the disabled flag?

Reply:
Can the ID be put in the plugin manager somewhere?

Reply:
You can look at the `id` in the `manifest.json` file. The id is also used for the folder name. So let's say we have ai plugin, and this is the manifest.json:

```json5
{
  "id": "bks-ai-shell",
  "author": "Beekeeper Studio",
   // other keys ...
}
```

That id is what we want to use inside the config files:

```ini
[plugins.bks-ai-shell]
enabled = false
```

The plugin is also located at the plugin id folder => `~/.config/beekeeper-studio/plugins/bks-ai-shell/`. Which is why `url.host` can also be used for identifying the plugin.